qt/component.cpp

```diff
@@ -394,11 +560,109 @@ QList<AppStream::Suggested> AppStream::Component::suggested() const
     return res;
 }
 
+void AppStream::Component::addSuggested(const AppStream::Suggested& suggested)
+{
+    as_component_add_suggested(m_cpt, suggested.suggested());
+}
+
+QStringList AppStream::Component::searchTokens() const
+{
+    valueWrap(as_component_get_search_tokens(m_cpt));
```

Comment:
Configure AppStream with `-DMAINTAINER=ON`, that will enable (almost) all compile-time maintainer checks.